src/DatasetBrowser/DatasetBrowserTable/index.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint no-console: 0 */
```

Comment:
could you remove these unnecessary eslint disable controls? Trying not to disable eslint if possible.

I think there are 3 of them

Reply:
i think these rule disables are necessary -- if we have clear error reporting in the console, it's easier to debug issues inside the callback chain in the future. some of the callbacks are punctuated with catch() blocks, and i'd like to report the error in human readable format in the console rather than discard it. i don't think console logs detract from the quality of an application -- they can be used to report client-side errors in a well-documented way. i also disabled the underscore-dangle rule, because it prevents the access of guppy properties that begin with underscores.